src/check/parse/IR.zig

```diff
@@ -1778,7 +1801,24 @@ pub const NodeStore = struct {
         statements: []const StatementIdx,
         /// The token that represents the newline preceding this block, if any
         whitespace: ?TokenIdx,
+
         region: Region,
+
+        pub fn toSExpr(self: @This(), gpa: Allocator, env: *base.ModuleEnv, ir: *IR) anyerror!sexpr.Node {
+            var children = std.ArrayList(sexpr.Node).init(gpa);
+
+            for (self.statements) |stmt_idx| {
+                const stmt = ir.store.getStatement(stmt_idx);
+                try children.append(try stmt.toSExpr(gpa, env, ir));
+            }
+
+            return .{
+                .node = .{
+                    .value = "body",
+                    .children = children.items,
```

Comment:
```suggestion
                    .children = try children.toOwnedSlice(),
```

This needs to be done for all uses of children. That or, even better, I would change children to simply be a `std.ArrayList` or even better a `std.ArrayListUnmanaged`